korangar/src/main.rs

```diff
@@ -331,11 +331,7 @@ fn main() {
         #[cfg(not(feature = "debug"))]
         let mut networking_system = NetworkingSystem::spawn();
         #[cfg(feature = "debug")]
-        let packet_callback = {
-            // SAFETY: This function leaks memory, but it's fine since we only call
-            // it once.
-            unsafe { interface::elements::PacketHistoryCallback::new() }
-        };
+        let packet_callback = interface::elements::PacketHistoryCallback::new();
```

Comment:
I don't love the fact that to the caller it looks like you are getting a new, unaliased `PacketHistoryCallback`. I was thinking of renaming the new function to something like `static_instance` or `get_singleton`. What do you think?

Reply:
Fine by me! How about `new_from_static_instace`?

Reply:
Hm, `get_static_instance`?

Reply:
Also fine!